src/components/Facility/DischargedPatientsList.tsx

```diff
@@ -293,6 +392,70 @@ const DischargedPatientsList = ({
               selected={qParams.ordering}
               onSelect={(e) => updateQuery({ ordering: e.ordering })}
             />
+            <div className="tooltip w-full md:w-auto" id="patient-export">
+              {!isExportAllowed ? (
+                <ButtonV2
+                  onClick={() => {
+                    advancedFilter.setShow(true);
+                    setTimeout(() => {
+                      const element =
+                        document.getElementById("bed-type-select");
+                      if (element)
+                        element.scrollIntoView({ behavior: "smooth" });
+                      Notification.Warn({
+                        msg: "Please select a seven day period.",
+                      });
+                    }, 500);
+                  }}
+                  className="mr-5 w-full lg:w-fit"
+                >
+                  <CareIcon icon="l-export" />
+                  <span className="lg:my-[3px]">Export</span>
+                </ButtonV2>
```

Comment:
We no longer use `ButtonV2` in newer codebases. Switch to using Button from shadcn